.github/scripts/build-binutils.sh

```diff
@@ -15,20 +17,25 @@ set -x # echo on
 mkdir -p $BUILD_PATH/binutils
 cd $BUILD_PATH/binutils
 
-echo "::group::Configure binutils"
-../../code/$BINUTILS_VERSION/configure \
-    --prefix=$INSTALL_PATH \
-    --target=$TARGET
-echo "::endgroup::"
+if [ $RUN_CONFIG = 1 ] ; then
```

Comment:
if config has never been run, maybe it makes sense to run it anyway or stop execution

Reply:
That's true. So far I left that decision to the user. Do you think it's worth to add detection whether the build was already configured here?

Reply:
I think it is worth to add at least a warning message and stop execution.